schema/deploy/function_opened_reporting_year.sql

```diff
@@ -7,8 +7,7 @@ returns ggircs_portal.reporting_year
 as $function$
   select *
   from ggircs_portal.reporting_year as _reporting_year
-  where ggircs_portal.current_timestamp() >= application_open_time
-  and ggircs_portal.current_timestamp() <= application_close_time;
+  where now() between application_open_time and application_close_time;
```

Comment:
https://wiki.postgresql.org/wiki/Don%27t_Do_This#Don.27t_use_BETWEEN_.28especially_with_timestamps.29

Usage of `between` is discouraged. This behaves as you would expect here since we're comparing timestamps with timestamps, but it could lead to bugs if we were to change `application_close_time` to `application_close_date`

Reply:
Oh, I had no idea. Thanks!